IOMC/EventVertexGenerators/src/HLLHCEvtVtxGenerator.cc

```diff
@@ -197,7 +197,7 @@ double HLLHCEvtVtxGenerator::integrandCC(double x, double z, double ct) const {
         std::exp(-ct * ct * ct * ct / sigs4 - z * z * z * z * cos2 * cos2 / sigs4 - 6 * ct * ct * z * z * cos2 / sigs4 -
                  sin2 / (4 * k2 * sigmax2) *
                      (2 + 4 * k2 * z * z - std::cos(2 * k * (z - ct)) - std::cos(2 * k * (z + ct)) -
-                      8 * k * s * std::cos(k * ct) * std::sin(k * z) - 4 * cosks * cosks * sinct * sinct)) /
+                      8 * k * CLHEP::s * std::cos(k * ct) * std::sin(k * z) - 4 * cosks * cosks * sinct * sinct)) /
```

Comment:
I was looking at  `if (!RF800) ` part of this code (our default variant) and #16942 (where this appeared first), and it's not obvious to me `second` here is right.
Hopefully "second" here is correct and not a typo z vs s.

Reply:
@slava77 , I suspect you are right CLHEP::s and CLHEP::second are the same things.

Reply:
I would agree that there is a mess of units in this code. If one is working within one unit system, for example CLHEP, everything is consistent and only IO needs to be checked. I would propose to open a new issue, which should require clean-up of this code, and left this PR unchanged.

Reply:
> I would propose to open a new issue, which should require clean-up of this code, and left this PR unchanged.

done in #44894